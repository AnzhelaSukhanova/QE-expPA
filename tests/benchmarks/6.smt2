(set-option :produce-models true)
(set-logic BV)
(declare-fun y () (_ BitVec 8))
(declare-fun z () (_ BitVec 8))
(assert
 (exists ((x (_ BitVec 8))) (and 
 (or (bvult x (bvadd (bvmul y #b00000101) #b00000111)) (= x (bvadd (bvmul y #b00000101) #b00000111))) 
 (or (bvult (bvmul (bvadd y z) #b00001000) x) (= (bvmul (bvadd y z) #b00001000) x)))))
