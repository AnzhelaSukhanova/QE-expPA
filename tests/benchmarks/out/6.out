(not (bvult (bvadd (bvmul y #x05) #x07) (bvmul (bvadd y z) #x08)))
